lib/blocks.js

```diff
@@ -350,6 +360,10 @@ BlockController.prototype.getPoolInfo = function(tx) {
     }
   }
 
+  if (tx.outputs.length > 1) {
+    return { "poolName": "P2Pool", "url": "https://vertcoin.org" };
```

Comment:
This is probably correct but IMO doesn't feel right to assume that multiple outputs must be P2Pool. I'm not sure if there's any other way to detect a P2Pool block though (@metalicjames it would be useful if P2Pool would put something in the coinbase like other pools)

It would be nice to be able to configure the pool name & url from pools.json somehow, maybe something like this could work?

```javascript
{
    "poolName": "P2Pool",
    "url": "https://scanner.vertcoin.org",
    "hasManyOutputs": true
}
```